src/experiments.js

```diff
@@ -54,6 +58,88 @@ export function isCanary(win) {
   return !!(win.AMP_CONFIG && win.AMP_CONFIG.canary);
 }
 
+/**
+ * Whether the specified experiment is on or off for origin trials. You should
+ * check if the experiment is already enabled before calling this function.
+ * @param {!Window} win
+ * @param {string} experimentId
+ * @param {!Object} opt_publicJwk Used for testing only.
+ * @return {!Promise<boolean>}
+ */
+export function isExperimentOnForOriginTrial(win, experimentId, opt_publicJwk) {
```

Comment:
@choumx Do you think there's a better way to provide the public key for testing besides the optional third param here?